lib/path.js

```diff
@@ -1342,11 +1342,10 @@ const posix = {
   },
 
 
-  _makeLong: function _makeLong(path) {
+  toLongUNCPath: function toLongUNCPath(path) {
```

Comment:
No, it's not a new API. It literally just renames the existing function and makes the old name a deprecated alias. This api is and has been used by core for a long time and changing its behavior is not something I want to do in this pr.